src/MicrosoftTeams.ts

```diff
@@ -209,6 +212,12 @@ namespace microsoftTeams {
 
         // Undocumented function used to clear state between unit tests
         this._uninitialize = () => {
+            if (frameContext) {
+                registerOnThemeChangeHandler(null);
```

Comment:
whoa, were we leaking before??

Reply:
This code only runs during unit testing and it was resulting in certain state to persist across test cases. It would not have had an impact on the use of our library in production.